src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -115,8 +118,26 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
   onDeleteClick,
   onCompleted,
 }) => {
-  const { state, goToFilters, dispatch, onComplete } = useAlertContext()
+  const {
+    state,
+    goToFilters,
+    dispatch,
+    onComplete,
+    createAlertError,
+  } = useAlertContext()
   const { jumpTo } = useJump()
+  const { sendToast } = useToasts()
+
+  const mWebView = window.innerWidth < 1280
+
+  useEffect(() => {
+    createAlertError &&
+      !mWebView &&
+      sendToast({
+        variant: "error",
+        message: t("common.errors.somethingWentWrong"),
```

Comment:
Do we still use i18next?

Reply:
I noticed that we use it in some places for error messages
But good question. Do you know if there was an agreement regarding i18?

Reply:
Not really, maybe something to search or ask for in #practice-web 🤔

Reply:
Uhu, will ask there

Reply:
And then will address all together if needed if it'a okay